osu.Game/Screens/Edit/Compose/Components/BeatDivisorControl.cs

```diff
@@ -279,6 +279,10 @@ protected override void OnDrag(DragEvent e)
                 handleMouseInput(e.ScreenSpaceMousePosition);
             }
 
+            protected override void OnDragEnd(DragEndEvent e)
+            {
+            }
```

Comment:
While the diagnosis that the problem lies with the base implementation of `OnDragEnd` seems correct, I'm not overly convinced that silencing the event outright is a better solution than just mirroring `OnDrag`:

```suggestion
            protected override void OnDragEnd(DragEndEvent e)
            {
                handleMouseInput(e.ScreenSpaceMousePosition);
            }
```